packages/@lwc/engine-core/src/3rdparty/snabbdom/snabbdom.ts

```diff
@@ -206,6 +215,27 @@ export function updateStaticChildren(parentElm: Node, oldCh: VNodes, newCh: VNod
     }
 }
 
+export function updateFakeSlotStaticChildren(
+    startElm: Node,
+    endElm: Node,
+    oldCh: VNodes,
+    newCh: VNodes
+) {
+    const parentElm = startElm.parentNode!;
+    const startIndex = Array.prototype.indexOf.call(parentElm, startElm);
```

Comment:
This should probably be:

```suggestion
    const startIndex = ArrayIndexOf.call(parentElm, startElm);
```

with

```js
import { ArrayIndexOf } from "@lwc/shared";
```